tests/projects/qt/quickplugin/xmake.lua

```diff
@@ -0,0 +1,12 @@
+add_rules("mode.debug", "mode.release")
+
+target("demo")
+    add_rules("qt.qmlplugin")
+    add_headerfiles("src/*.h")
+    add_files("src/*.cpp")
+
+    if is_plat("windows") then
+        add_cxxflags("/permissive-")
```

Comment:
If qt has to rely on this compile option, then you can set it in qt/env/xmake.lua. If it is only an optional option, then remove it here. Leave it to the user to set it additionally as needed.